sdk/azidentity/client_certificate_credential_test.go

```diff
@@ -90,27 +90,15 @@ func TestClientCertificateCredential_GetTokenSuccess(t *testing.T) {
 	}
 }
 
-func TestClientCertificateCredential_GetTokenSuccess_withCertificateChain(t *testing.T) {
```

Comment:
Was this a duplicate test?

Reply:
Yes. The one below it tests the same path and, with the update in this PR, is more comprehensive